semcore/date-picker/src/index.d.ts

```diff
@@ -306,6 +313,14 @@ declare const InputTrigger: Intergalactic.Component<
     Indicator: typeof Input.Addon;
     MaskedInput: Intergalactic.Component<'input', InputMaskValueProps & DatePickerMaskedInputProps>;
   };
+};
+
+declare const RangeInputTrigger: Intergalactic.Component<
```

Comment:
Great! It's so cool that now it's impossible to write `DatePicker.Trigger.DateRange` or `DateRangePicker.Trigger.SingleDateInput`. I love it.